docs/intro/glossary.md

```diff
@@ -0,0 +1,296 @@
+
+### Timeseries ID
+
+Same as [time series ID](#time-series-id).
+
+## Metrics
+
+All metrics related to power, energy, current, etc. use the [PSC](#psc) to determine the sign of the value.
+
+### Consumption
+
+The amount of electricity flowing into a [component](#component). It is the clipped positive value when using the [PSC](#psc), so if the electricity is flowing out of the component instead, this will be zero.
+
+### Gross Consumption
+
+[Consumption](#consumption) before accounting for any local generation from [solar](#solar-panel), [wind](#wind-turbine) or [CHP](#chp-plant).
+
+### Instantaneous Power
+
+Same as [power](#power).
+
+### Load
+
+Typically refers to a device that [consume](#consumption) electricity, but also to the amount of electricity [consumed](#consumption) by such a device.
+In a [microgrid](#microgrid) context, it is often used to refer to all the electrical devices that are doing active work, for example a light bulb, a motor, a cooler, etc.
+
+When using the [PSC](#psc), this is the same as [consumption](#consumption) and it is a positive value. If a *load* [generates](#production) electricity instead, it is a negative value but it wouldn't typically be called a *load*.
+
+### Net Consumption
+
+This term traditionally refers to the difference between the [gross consumption](#gross-consumption) and the local generation (like [PV](#pv) [production](#production)). It is the electricity [consumption](#consumption) that needs to be provided by the [battery](#battery) or from the public [grid](#grid).
+
+### Net Load
+
+Same as [net consumption](#net-consumption).
+
+### Power
+
+The rate of energy transfer, i.e. the amount of energy transferred per unit of time. It is typically measured in Watts (W).
+
+For [AC](#ac) electricity, there are three types of power: [active](#active-power), [reactive](#reactive-power), and [apparent](#apparent-power) (P, Q and |S| respectively in [power triangle](#power-triangle)).
+
+See the [Wikipedia article](https://en.wikipedia.org/wiki/AC_power) for more information.
+
+#### Power Triangle
+
+The visual representation of the relationship between the three types of [power](#power) and the [phase](#phase).
+
+![Power triangle](../img/power-triangle.svg)
+
+([CC-BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/deed.en), [Wikimedia Commons](https://commons.wikimedia.org/wiki/File:Cmplxpower.svg))
+
+#### Active Power
+
+The power that is actually consumed by the [load](#load). It is the real part of the [apparent power](#apparent-power), P in the [power triangle](#power-triangle).
+
+#### Apparent Power
+
+The power that is actually supplied to the [load](#load). The magnitude of the vector sum of the [active](#active-power) and [reactive](#reactive-power) power, |S| in the [power-triangle](#power-triangle).
+
+#### Phase
+
+The angle of difference (in degrees) between current and voltage in an [AC](#ac) circuit, φ in the [power triangle](#power-triangle).
```

Comment:
Wikipedia. https://en.wikipedia.org/wiki/AC_power

Reply:
Well, it says "Phase of voltage relative to current", I read "Phase **or** voltage relative to current" when I was writing it, maybe that makes a difference.

Reply:
But in wikipedia it is unambiguous. There it is called `Phase of voltage relative to current`. Here it is ambiguous.

Reply:
Also, conventionally, systems use [Power Factor](https://en.wikipedia.org/wiki/Power_factor), instead of `φ`.

Reply:
Or maybe I just remove it if it's not very relevant, I just added to make the several components of AC power more complete.

Reply:
Sure, on it.

Reply:
Since I started the review on just one line, I cannot add multi-line suggestion I think :(
I have typed up a suggestion here, if it helps:

```md
#### Phase of Voltage Relative to Current

The angle of difference (in degrees) between current and voltage in an [AC](#ac) circuit, φ in the [power triangle](#power-triangle). Applicable to AC electricity only.

#### Power Factor

The ratio of real power to apparent power. Applicable to AC electricity only. Can be obtained by computing the cosine of  [φ](#phase-of-voltage-relative-to-current), in the [power triangle](#power-triangle). See the [Wikipedia article](https://en.wikipedia.org/wiki/Power_factor) for more details.
```

Reply:
>Or maybe I just remove it if it's not very relevant, I just added to make the several components of AC power more complete.

IMO it makes sense to keep it, since it's used in the diagram. But your call.

Reply:
Great, thanks! I applied it but changed the `Applicable to AC electricity only` in **Power Factor** to ` in an [AC](#ac) circuit` as it is shorter and matches **Phase of Voltage Relative to Current**.

I also added AC to other AC definitions.

Please see:
* 63a5fbfc059e4bfe7b118d5d22089b296d66e9dc